### ISSUE TITLE
Installing 0.12.3 Yields "Table mydatabase.myprefix_meta doesn't exist" on Fresh Install

### ISSUE THREAD

**Issue by KermMartian** [anchorcms/anchor-cms#1213]
### Summary

After unpacking Anchor, discovering that Composer was required, and going through the installation steps, the frontpage did not work, and I discovered no tables had actually been created. I replicated this with a database user with only create, select, insert, update, and delete, as well as a database user with all permissions on the given database.

### Expected Behaviour

- Unzip release 0.12.3
- Chmod the required directories as indicated
- Download composer
- Run `composer install`
- Delete composer
- Create database and database user via MySQL CLI
- Visit the installer, complete installation process
- See live, functional site

### Actual Behaviour

- Performed all of the above, but on the last step, saw the error message in the title in the browser instead of a shiny new blog.

### Context details (if applicable)

- Anchor version: 0.12.3
- Server setup: Debian, PHP 5.6.3
- URL: n/a

**Comment by TheBrenny** [comment]
Awesome detail for the issue! Thanks heaps!

Just a few questions to help diagnose this issue:
1) Under "Actual Behaviour", you identify "all of the above" - I'm assuming you're referring to the "Summary"?
2) Are you able to provide screenshots of your PHPMyAdmin/MySQL CLI?

I'll pull the repo and test it out - possibly could've been caused by #1212, or if it was merged after you pulled the repo, maybe you didn't get the patch?

**Comment by KermMartian** [comment]
I grabbed the release zip build from the top of the Releases page, actually. Also, I should clarify that I mean that I performed all those steps, and the installer /appeared/ to work fine, but on the last step, I saw the indicated error message instead of my new blog index.

**Comment by TheBrenny** [comment]
Aah, I see. Are you able to grab the zip from the master? This issue has been fixed with #1212. I'll leave this open pending your findings, and I'll fix up the zip on the release page.

**Comment by KermMartian** [comment]
With the latest master zip, I receive the following error message on the blog homepage after installing (progress, at least!):

```
Database Error: SQLSTATE[42000]: Syntax error or access violation: 1067 Invalid default value for 'updated'

SQL: ALTER TABLE `anchor_posts` ADD COLUMN `updated` DATETIME DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP AFTER `created`
The error has been logged in /anchor/errors.log
```

**Comment by TheBrenny** [comment]
Unfortunately, the column is of `DATETIME` type, rather than a `TIMESTAMP` type. `... DATETIME DEFAULT CURRENT_TIMESTAMP ...` is only available in MySQL 5.6+.

**Comment by KermMartian** [comment]
Aha, thanks for that clarification. May I recommend an update to https://anchorcms.com/docs , indicating the updated PHP version and MySQL version required, as well as the fact that Composer is also now a requirement? Thanks in advance!

**Comment by crshd** [comment]
Same thing happens on my dev-machine. Installed with `composer create-project anchorcms/anchor-cms anchor`

Environment:
```
macOS 10.12.4
PHP 7.1.6 (cli) (built: Jun 23 2017 08:42:14) ( NTS )
mysql  Ver 15.1 Distrib 10.1.22-MariaDB
```

Manually cloning master and installing deps works fine.

**Comment by TheBrenny** [comment]
@crshd Just confirming by "same thing", you mean that you used composer to install the project, ran through the installation process of Anchor, and nothing popped up in the DB? But when grabbing the zip of Anchor from GH, it all works? This really is a strange one. It might have something to do with the way that Composer/Packagist gets the project? @CraigChilds94 might know more on this topic?

**Comment by crshd** [comment]
Exactly. After creating a project with `composer create-project` and running the installer, the database was completely empty. Cloning master and running the installer did populate the DB.

Anchor seems to be running fine on MariaDB. I haven't noticed anything going wrong so far.

**Comment by evansharp** [comment]
I think we should drop support for MySQL < 5.6. See my comment on

**Comment by 14u2ponder** [comment]
Clarifying: is 12.3 a full install or just an update? Issues: 12.3 does not come with composer vendor folder, and I was never able to install it. Installed 12.1 fine. My question is, do I just replace system folders from 12.1 to 12.3 if I want 12.3 functions? Is that it?

**Comment by TheBrenny** [comment]
Closed as this was a duplicate of #1205.

@14u2ponder, that *SHOULD* work... Try following the setup instructions contained within the Readme.
